islandora_pathauto.module

```diff
@@ -112,7 +112,9 @@ function islandora_pathauto_create_alias($object, $op) {
  * Implements hook_islandora_object_ingested().
  */
 function islandora_pathauto_islandora_object_ingested($object) {
-  islandora_pathauto_create_alias($object, 'insert');
+  if( !path_load('islandora/object/' . $object -> id) ){
```

Comment:
The member operator (->) shouldn't be space separated.  Also, watch your `if` statements throughout.  Always a space between the control keyword and the parenthesis, then another space between the close parenthesis and the curly brace.
https://www.drupal.org/coding-standards#controlstruct
When submitted to Islandora, this code will be run through CodeSniffer and it will be very picky.

I'm not sure if there is CodeSniffer policy on the space between the open parenthesis and the conditional, but the documentation seems to consistently omit it.